lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -154,8 +155,7 @@ def managed_disk_to_dict(managed_disk):
         location=managed_disk.location,
         tags=managed_disk.tags,
         create_option=create_data.create_option.value.lower(),
-        source_uri=create_data.source_uri,
-        source_resource_uri=create_data.source_resource_id,
+        source_uri=create_data.source_uri or create_data.source_resource_id,
```

Comment:
is this behavior change in sdk  that source_uri same meaning with source_resource_id?

Reply:
There is a request from Matt to collapse these two options.